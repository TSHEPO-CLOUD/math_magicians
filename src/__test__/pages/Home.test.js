import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../pages/Home';

it('should match the snapshot of Home', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});