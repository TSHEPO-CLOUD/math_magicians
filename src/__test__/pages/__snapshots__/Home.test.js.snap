// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot of Home 1`] = `
<div
  className="home-holder"
>
  <h2
    className="home-title"
  >
    Welcome to my app!
  </h2>
  <p
    className="home-text"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi.
  </p>
</div>
`;
